librz/egg/egg.c

```diff
@@ -175,6 +186,9 @@ RZ_API bool rz_egg_setup(RzEgg *egg, const char *arch, int bits, int endian, con
 	} else {
 		return false;
 	}
+	if (egg->remit && egg->remit->begin) {
+		egg->remit->begin(egg);
+	}
```

Comment:
I think you misunderstood me.
We use `init` and `fini` as constructors and destructor functions. but here `init` is used to start the creation of the assembly file. i wanted you to convert the original `init` function into `begin` so you would free up the `init` method to be the real constructor function which we did not have before.